credit_calculator.py

```diff
@@ -28,16 +28,15 @@ def months_until_paid_out(credit_sum, monthly_payment, initial_payment, monthly_
 
 if __name__ == "__main__":
     
-    data=[]
-    for arg in sys.argv:
-        data.append(arg)
-        
-    credit_sum = float(data[1])
-    monthly_payment = float(data[2])
-    
+    parser = argparse.ArgumentParser(description='Arguments required to do the calculations')
+    parser.add_argument('credit_sum', type=float, help='Credit Summary')
```

Comment:
`Credit summary` -> `Credit sum` (It changes the sense)

Also, it may be feel like a proper command line tool if do `--sum`, instead of `credit_sum` in `add_argument`. The same can be with `monthly_payment` -> `--payment` or `-p`

Reply:
I changed the names in the help parameter of add_argument but I'm not sure if it's where you wanted it.